plugins/core/base/routes/tool-default.js

```diff
@@ -66,11 +66,24 @@ module.exports = {
           params: {
             tool: Joi.string().required(),
             path: Joi.string().required()
+          },
+          query: {
+            appendItemToPayload: Joi.string().optional()
+          },
+          options: {
+            allowUnknown: true
           }
         }
       },
       handler: async (request, h) => {
-        return await Reflect.apply(handler, this, [options, request, h]);
+        let payload = null;
```

Comment:
is it easier if we pass null if no item is appended to payload? Or could we just leave it as undefined?

Reply:
the handler currently checks for `null` and sends a GET request to the tool if there is no payload. I like to have `null` as explicit null that is on purpose instead of `undefined` which could just be a mistake in accessing a property or something.

Reply:
fair point 👍